advanced/image_processing/examples/plot_face_tv_denoise.py

```diff
@@ -2,18 +2,19 @@
 Total Variation denoising
 ===========================
 
-This example demoes Total-Variation (TV) denoising on Lena.
+This example demoes Total-Variation (TV) denoising on a Face.
 """
 
 import numpy as np
 import scipy
+import scipy.misc
 import matplotlib.pyplot as plt
-from skimage.filter import denoise_tv_chambolle
+from skimage.restoration import denoise_tv_chambolle
```

Comment:
We decided to postponed that change (filter -> restoration) to still support old versions of scikit-image. In a later release of the scipy lectures, we'll change it.